package.json

```diff
@@ -35,6 +35,7 @@
     "test:nyc:lcov": "nyc report --reporter=text-lcov > .lcov.info",
     "test:nyc:report": "nyc report",
     "test:repoutils": "yarn workspace botbuilder-repo-utils test",
+    "test:schemas": "mocha \"libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js\" --exit",
```

Comment:
This test runs automatically during the normal `test:github` and `test:devops` scripts, correct? What's the reasoning for having a separate test script.

Reply:
Currently, they're separate. These two little tests require an additional dependency (bf-cli) and take ~40 seconds to run. It seemed like a lot of additional overhead to make it run every time. I can make them run along with everything, if you'd prefer.